modules/openapi-generator/src/main/resources/typescript-fetch/api.mustache

```diff
@@ -1,364 +1,442 @@
-/// <reference path="./custom.d.ts" />
 // tslint:disable
 {{>licenseInfo}}
 
-import * as url from "url";
-import * as portableFetch from "portable-fetch";
-import { Configuration } from "./configuration";
+export const BASE_PATH = "{{{basePath}}}".replace(/\/+$/, "");
 
-const BASE_PATH = "{{{basePath}}}".replace(/\/+$/, "");
-
-/**
- *
- * @export
- */
-export const COLLECTION_FORMATS = {
-    csv: ",",
-    ssv: " ",
-    tsv: "\t",
-    pipes: "|",
-};
+{{#models}}
+{{#model}}{{#isEnum}}{{>modelEnum}}{{/isEnum}}{{^isEnum}}{{>modelGeneric}}{{/isEnum}}{{/model}}
+{{/models}}
 
-/**
- *
- * @export
- * @interface FetchAPI
- */
-export interface FetchAPI {
-    (url: string, init?: any): Promise<Response>;
+{{#apiInfo}}
+{{#apis}}
+{{#operations}}
+{{#operation}}
+{{#allParams.0}}
+export interface {{operationIdCamelCase}}Request {
```

Comment:
@TiFu this fork also generates interfaces for all API parameters. This is very helpful as a consumer because it allows you to reference them in your own methods/variables. The original generator uses inline type signatures which aren't reusable.